test/page.spec.js

```diff
@@ -340,6 +340,18 @@ module.exports.addTests = function({testRunner, expect, puppeteer, DeviceDescrip
     });
   });
 
+  describe('Page.Events.Log.entryAdded', function() {
+    it('should trigger correct Log', async({page, server}) => {
+      await page.goto('about:blank');
+      let message;
+      page.on('console', event => message = event);
+      page.evaluate(async() => fetch('https://example.org/').catch(e => {}));
```

Comment:
nit: let's use `server.EMPTY_PAGE`instead of `example.com`

Reply:
DOne